recipes/pear.rb

```diff
@@ -7,26 +7,17 @@
 
 include_recipe 'php'
 
-# PHP Extension and Application Repository PEAR channel
-pearhub_chan = php_pear_channel 'pear.php.net' do
-  action :update
-end
-
 # upgrade PEAR
 php_pear 'PEAR' do
-  channel pearhub_chan.channel_name
+  channel 'pear.php.net'
   action :upgrade
 end
 
-# PHPUnit PEAR channel
-php_pear_channel 'pear.phpunit.de' do
-  action :discover
-end
-
 # install/upgrade phpcs
 package = 'PHP_CodeSniffer'
 
 # upgrade when package is installed and latest version is required
+# TODO: This throws an error on first run, because pear is not installed yet
```

Comment:
Yeah, send me a gist or pastebin for it. I probably won't have time today, but will do my best over the next few days... Thanks in advance!